install/install-by-deb.md

```diff
@@ -1,54 +1,65 @@
-# Installing Fluentd Using deb Package
+# Installing Fluentd Using `deb` Package
 
-This article explains how to install the
-[td-agent](https://docs.treasuredata.com/articles/td-agent) deb package,
-the stable Fluentd distribution package maintained by [Treasure Data, Inc](http://www.treasuredata.com/).
+This article explains how to install `td-agent` deb package, the stable Fluentd
+distribution package maintained by [Treasure Data,
+Inc](http://www.treasuredata.com/).
 
 
-## What is td-agent?
+## What is `td-agent`?
 
-Fluentd is written in Ruby for flexibility, with performance sensitive
-parts written in C. However, some users may have difficulty installing
-and operating a Ruby daemon.
+Fluentd is written in Ruby for flexibility, with performance-sensitive parts in
+C. However, some users may have difficulty installing and operating a Ruby
+daemon.
 
-That's why [Treasure Data, Inc](http://www.treasuredata.com/) is
-providing **the stable distribution of Fluentd**, called `td-agent`. The
-differences between Fluentd and td-agent can be found
-[here](https://www.fluentd.org/faqs).
-
-This installation guide is for td-agent v3/td-agent v4.
-td-agent v3/v4 uses fluentd v1 in the core. See [this page](/overview/td-agent-v2-vs-v3-vs-v4.md) for the comparison and supported OS.
+That is why [Treasure Data, Inc](http://www.treasuredata.com/) provides **the
+stable distribution of Fluentd**, called `td-agent`. The differences between
+Fluentd and `td-agent` can be found [here](https://www.fluentd.org/faqs).
 
+This installation guide is for `td-agent` v3/v4.
+`td-agent` v3/v4 uses fluentd v1.0 in the core. See
+[this page](/overview/td-agent-v2-vs-v3.md) for the comparison and supported OS.
 
 ## Step 0: Before Installation
 
-Please follow the [Preinstallation Guide](/install/before-install.md) to configure
-your OS properly. This will prevent many unnecessary problems.
+Please follow the [Pre-installation Guide](/install/before-install.md) to
+configure your OS properly.
 
 
 ## Step 1: Install from Apt Repository
 
-It's HIGHLY recommended that you set up **ntpd** on the node to prevent
-invalid timestamps in your logs. Please check the [Preinstallation Guide](/install/before-install.md).
+For Ubuntu, the supported versions are:
+
+-   Ubuntu 18.04 LTS / Bionic 64bit
```

Comment:
Remove these supported OS because hard to sync with https://docs.fluentd.org/quickstart/td-agent-v2-vs-v3-vs-v4#supported-platforms table. Debian too.